media/baseset/openttd/overlay_rocks.nfo

```diff
@@ -0,0 +1,102 @@
+// See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+//
+   -1 * 0	 0C "Overlay rocks"
+   -1 * 3	 05 1A 5F
+   -1 sprites/overlay_rocks.png 8bpp    1 1  64  31 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp   81 1  64  31 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  161 1  64  23 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  241 1  64  23 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  321 1  64  31 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  401 1  64  31 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  481 1  64  23 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  561 1  64  23 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  641 1  64  39 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp  721 1  64  39 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp  801 1  64  31 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp  881 1  64  31 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp  961 1  64  39 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp 1041 1  64  39 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp 1121 1  64  31 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp 1201 1  64  47 -31  -16 normal
+   -1 sprites/overlay_rocks.png 8bpp 1281 1  64  15 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp 1361 1  64  31 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp 1441 1  64  31 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp    1 1  64  31 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp   81 1  64  31 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  161 1  64  23 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  241 1  64  23 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  321 1  64  31 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  401 1  64  31 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  481 1  64  23 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  561 1  64  23 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp  641 1  64  39 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp  721 1  64  39 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp  801 1  64  31 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp  881 1  64  31 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp  961 1  64  39 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp 1041 1  64  39 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp 1121 1  64  31 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp 1201 1  64  47 -31  -16 normal
+   -1 sprites/overlay_rocks.png 8bpp 1281 1  64  15 -31    0 normal
+   -1 sprites/overlay_rocks.png 8bpp 1361 1  64  31 -31   -8 normal
+   -1 sprites/overlay_rocks.png 8bpp 1441 1  64  31 -31   -8 normal
```

Comment:
Any reason for this to exist twice? I'm not seeing that the first set is used in the code.

Reply:
"Maybe". Initial revision had just two sets "overlay rocks" and "snowy overlay rocks", with the first set being used for the first snow layer. I then figured that as rocks already have snow density, having 4 sets of snowy overlay rocks would be simple enough to add on.

I left the original "overlay rocks" set as is, because people were making noises about having desert rocks. Not a brilliant reason but there we go.